migration/5_imp_smime_extrakeys.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Create IMP SMIME tables.
+ *
+ * Copyright 2010-2017 Horde LLC (http://www.horde.org/)
+ *
+ * See the enclosed file LICENSE for license information (GPL). If you
+ * did not receive this file, see http://www.horde.org/licenses/gpl.
+ *
+ * @author   Rafael te Boekhorst <boekhorst@b1-systems.de>
+ * @category Horde
+ * @license  http://www.horde.org/licenses/gpl GPL
+ * @package  IMP
+ */
+class ImpSmimeExtraKeys extends Horde_Db_Migration_Base
```

Comment:
Do we need another migration for that?

If you only used this for you dev environment, can you not put that into the `4_imp_smime.php` file as well?

Reply:
yes I will do that. I was curious to try it out I think :D